scripts/tikv_details.json

```diff
@@ -19248,4 +19248,4 @@
   "title": "Test-Cluster-TiKV-Details2",
```

Comment:
`Test-Cluster-TiKV-Details2` needs to be changed to `Test-Cluster-TiKV-Details`.

Reply:
This is a cherry-pick. I think it should exclude extra fixes.